exoctk/utils.py

```diff
@@ -34,28 +35,113 @@
 ON_TRAVIS_OR_RTD = HOME_DIR == '/home/travis' or HOME_DIR == '/Users/travis' or HOME_DIR == '/home/docs'
 if not ON_TRAVIS_OR_RTD:
     if not EXOCTK_DATA:
-        raise ValueError(
-            'The $EXOCTK_DATA environment variable is not set.  Please set the '
-            'value of this variable to point to the location of the ExoCTK data '
+        print(
+            'WARNING: The $EXOCTK_DATA environment variable is not set.  Please set the '
+            'value of this variable to point to the location of the exoctk_data '
             'download folder.  Users may retreive this folder by clicking the '
-            '"ExoCTK Data Download" button on the ExoCTK website.'
+            '"ExoCTK Data Download" button on the ExoCTK website, or by using '
+            'the exoctk.utils.download_exoctk_data() function.'
         )
 
     # If the variable exists but doesn't point to a real location
     if not os.path.exists(EXOCTK_DATA):
-        raise FileNotFoundError(
-            'The $EXOCTK_DATA environment variable is set to a location that '
+        print(
+            'WARNING: The $EXOCTK_DATA environment variable is set to a location that '
             'cannot be accessed.')
 
     # If the variable exists, points to a real location, but is missing contents
-    for item in ['modelgrid', 'fortney', 'exoctk_log', 'generic']:
+    for item in ['modelgrid', 'fortney', 'exoctk_log', 'generic', 'groups_integrations']:
         if item not in [os.path.basename(item) for item in glob.glob(os.path.join(EXOCTK_DATA, '*'))]:
-            raise KeyError('Missing {}/ directory from {}'.format(item, EXOCTK_DATA))
+            print(
+                'WARNING: Missing {}/ directory from {}. Please ensure that the ExoCTK data package has been '
+                'downloaded. Users may retrieve this package by clicking the "ExoCTK Data Donwload" '
+                'button on the ExoCTK website, or by using the exoctk.utils.download_exoctk_data() '
+                'function'.format(item, EXOCTK_DATA))
 
 MODELGRID_DIR = os.path.join(EXOCTK_DATA, 'modelgrid/')
 FORTGRID_DIR = os.path.join(EXOCTK_DATA, 'fortney/')
 EXOCTKLOG_DIR = os.path.join(EXOCTK_DATA, 'exoctk_log/')
 GENERICGRID_DIR = os.path.join(EXOCTK_DATA, 'generic/')
+GROUPS_INTEGRATIONS_DIR = os.path.join(EXOCTK_DATA, 'groups_integrations/')
```

Comment:
needs another line for `exoctk_contam` folder:
`CONTAM_DIR = os.path.join(EXOCTK_DATA, 'exoctk_contam/')`